docs/api/crash-reporter.md

```diff
@@ -119,6 +119,14 @@ Remove a extra parameter from the current set of parameters so that it will not
 
 See all of the current parameters being passed to the crash reporter.
 
+### `crashReporter.getCrashesDirectory()`
+
+Returns `String` - The directory where crashes are temporarily stored before being uploaded.
+
+### `crashReporter.getProductName()`
```

Comment:
I am not sure we want to make this a public API

Reply:
I think it was only mean to be used by the tests

Reply:
miniak  u are great am new here thx for ur help >  that guys name nornagon and Shelley Vohr Jeremy Apthorp and one  googleapis/nodejs remove my job coz its there pic and madia voice ... i dont know how stop that can u help me .. u can check main and see

Reply:
In that case can we delete the function? It doesn't seem to be used by anything...

Reply:
I would delete them